tests/cases/fourslash/quickInfoSpecialPropertyAssignment.ts

```diff
@@ -3,8 +3,10 @@
 ////class C {
 ////    constructor() {
 ////      /** Doc */
-////      this./**/x = 0;
+////      this./*write*/x = 0;
+////      this./*read*/x;
 ////    }
 ////}
 
-verify.quickInfoAt("", "(property) C.x: number", "Doc");
+verify.quickInfoAt("write", "(property) C.x: any", "Doc");
+verify.quickInfoAt("read", "(property) C.x: number", "Doc");
```

Comment:
The desired behavior here seems debatable to me—I think this `any` would have confused me before I learned about some similar quirks around quick info on assignments to evolving array types. There’s some sense in which hovering a variable to get quick info always feels like _reading_, even if the location is actually a write.

Reply:
yeah but then we cant distinguish between getter or setter anytime.. so.. shrug.. i think type at that location seems like better than always assuming read which is what this PR does..